stetho/src/main/java/com/facebook/stetho/inspector/network/ResponseBodyFileManager.java

```diff
@@ -56,19 +73,44 @@ public ResponseBodyData readFile(String requestId) throws IOException {
       }
       ResponseBodyData bodyData = new ResponseBodyData();
       bodyData.base64Encoded = firstByte != 0;
-      bodyData.data = readContentsAsUTF8(in);
+
+      if (mRequestIdMap.containsKey(requestId)) {
+        AsyncPrettyPrinter asyncPrettyPrinter = mRequestIdMap.get(requestId);
+        bodyData.data = prettyPrintContentWithTimeOut(asyncPrettyPrinter, in);
+      } else {
+        bodyData.data = readContentsAsUTF8(in);
+      }
       return bodyData;
     } finally {
       in.close();
     }
   }
 
-  private static String readContentsAsUTF8(InputStream in) throws IOException {
+  public static String readContentsAsUTF8(InputStream in) throws IOException {
     ByteArrayOutputStream out = new ByteArrayOutputStream();
     Util.copy(in, out, new byte[1024]);
     return out.toString("UTF-8");
   }
 
+  private String prettyPrintContentWithTimeOut(
+      AsyncPrettyPrinter asyncPrettyPrinter,
+      InputStream in) throws IOException {
+    AsyncPrettyPrintingCallable prettyPrintingCallable = new AsyncPrettyPrintingCallable(
+        in,
+        asyncPrettyPrinter);
+    Future<String> future = sExecutor.submit(prettyPrintingCallable);
+    try {
+      return Util.getUninterruptibly(future, 10, TimeUnit.SECONDS);
```

Comment:
In general, always declare your constants as named variables.  In this case, with `private static final ... ` in the top of the class.  Likewise, below, replace your repeated use of the literal with the named constant so that there is only one place to change to keep the code correct and consistent.